### DISCUSSION TITLE
Alguien más no le han aceptado aun en el grupo de teams de la mision backend?

### DISCUSSION THREAD

**Discussion by maxjhael** [Launch-X-Latam/MisionFrontEnd#500] [Team Posts]
Llevo esperando y nada, y quiero ver la clase de ayer

**Comment by VAlonzoGO** [comment]
ya intentaste dando click en el link de la reunión de ayer yo asi veía con las que me atrase en el modulo pasado

**Comment by maxjhael** [comment]
Si se que se puede hacer eso, el problema es que aun esta pendiente mi solicitud y es molesto perderse una clase por algo que no esta en manos de uno

**Comment by DanielBustillosVillar** [comment]
A mi tampoco me han aceptado todavía :,(

**Comment by Ricardo-Vargas-Gonzalez** [comment]
Deberían volver a enviar la solicitud, o revisar si la enviaron bien, yo la envié hoy y me aceptaron hoy mismo. ;)

**Comment by gersondlarosa** [comment]
A mi no me llego como hago la petición para ver la clase del día de ayer
